### PR TITLE
added pagination

### PR DESCRIPTION
**Pull request by Awesomeasma**
### Description

(Write your answer here.)

### Checklist

- [ ] I am making a proper pull request, not spam.
- [ ] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

(Write your answer here.)

## Add relevant screenshot or video (if any)

(Add here)

### CODE REVIEW

**Review by wasimreja — CHANGES_REQUESTED** [review]
@Awesomeasma great work. it would be better if the pagination shows after you search for a book. and enhance pagination design.
https://codepen.io/MarioD/pen/OmWaqz

**Comment by vk20july** [comment]
@wasimreja sir but this is unfair for me as you assigned this task to me and i have also made a PR for this

**Comment by Awesomeasma** [comment]
@wasimreja 
I have updated it. Please review.

File: css/style.css [hunk 1]
```diff
@@ -1020,4 +1020,170 @@ footer {
...
+
+.transition-next .little-dot--last {
+  animation: slideRight 0.4s 0.3s forwards cubic-bezier(0.165, 0.84, 0.44, 1);
+}
+
+.big-dot {
+  width: 12px;
+  height: 12px;
+  border-radius: 100%;
+  background: #f6af54;
```
Comment (wasimreja) [inline css/style.css:1124]:
```suggestion
  background: #dbd3d3;
```

File: css/style.css [hunk 2]
```diff
@@ -1020,4 +1020,170 @@ footer {
...
+
+.transition-next .big-dot-container {
+  animation: big-dot-container--animation-next 0.3s forwards;
+}
+
+.btn {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  fill: #f6af54;
```
Comment (wasimreja) [inline css/style.css:1174]:
```suggestion
  fill: #dbd3d3;
```

File: css/style.css [hunk 3]
```diff
@@ -1020,4 +1020,170 @@ footer {
...
+  cursor: pointer;
+  transition: opacity 0.2s;
+}
+
+.btn:hover {
+  opacity: 0.6;
```
Comment (wasimreja) [inline css/style.css:1180]:
```suggestion
  fill: #d0c5c5;
```

**Review by wasimreja — CHANGES_REQUESTED** [review]
@Awesomeasma pagination is not working and the button is overlapping. fix this. add padding to the bottom. 
Screenshot 2023-03-08 at 21-48-35 Book Finder

File: css/style.css [hunk 4]
```diff
@@ -1020,4 +1020,170 @@ footer {
...
+}
+
+.transition-next .pagination-container {
+  animation: pagination-container--animation-next 0.3s forwards;
+}
+
+.little-dot {
+  width: 6px;
+  height: 6px;
+  background: #dc3545;
```
Comment (wasimreja) [inline css/style.css:1057]:
```suggestion
  background: #dbd3d3;
```

File: css/style.css [hunk 5]
```diff
@@ -1020,4 +1020,170 @@ footer {
...
+
+.transition-next .little-dot--last {
+  animation: slideRight 0.4s 0.3s forwards cubic-bezier(0.165, 0.84, 0.44, 1);
+}
+
+.big-dot {
+  width: 12px;
+  height: 12px;
+  border-radius: 100%;
+  background: #dbd3d3;
```
Comment (wasimreja) [inline css/style.css:1124]:
```suggestion
  background: #eba544;
```

**Review by wasimreja — CHANGES_REQUESTED** [review]

**Comment by Awesomeasma** [comment]
> @Awesomeasma pagination is not working and the button is overlapping. fix this. add padding to the bottom. you have put .vscode file to css. delete this. Screenshot 2023-03-08 at 21-48-35 Book Finder

I apologize sir, I have updated the css & other components, I am fixing the functionality of the pagination buttons, I'll get it done, soon.

**Comment by Awesomeasma** [comment]
> @Awesomeasma pagination is not working and the button is overlapping. fix this. add padding to the bottom. you have put .vscode file to css. delete this. Screenshot 2023-03-08 at 21-48-35 Book Finder

Please check once..

**Comment by wasimreja** [comment]
@Awesomeasma fix the functionality of the pagination. your branch has conflicts. resolve this. rest is okay.

**Comment by Awesomeasma** [comment]
> @Awesomeasma fix the functionality of the pagination. your branch has conflicts. resolve this. rest is okay.

Please tell me how do i fix it, I am trying to do so, but i am not able to.

**Comment by wasimreja** [comment]
> > @Awesomeasma fix the functionality of the pagination. your branch has conflicts. resolve this. rest is okay.
> 
> Please tell me how do i fix it, I am trying to do so, but i am not able to.

dm me on discord.

**Pull request closed without merging**